internal/namespaces/instance/v1/custom_server_create.go

```diff
@@ -231,22 +237,31 @@ func instanceServerCreateRun(ctx context.Context, argsI interface{}) (i interfac
 		serverReq.Image = args.Image
 	}
 
-	getImageResponse, err := apiInstance.GetImage(&instance.GetImageRequest{
-		Zone:    args.Zone,
-		ImageID: serverReq.Image,
-	})
-	if err != nil {
-		logger.Warningf("cannot get image %s: %s", serverReq.Image, err)
-	}
+	var (
+		getImageResponse *instance.GetImageResponse
+		serverType       *instance.ServerType
+	)
+	if args.Image != "none" {
+		getImageResponse, err := apiInstance.GetImage(&instance.GetImageRequest{
+			Zone:    args.Zone,
+			ImageID: serverReq.Image,
+		})
+		if err != nil {
+			logger.Warningf("cannot get image %s: %s", serverReq.Image, err)
+		}
 
-	serverType := getServerType(apiInstance, serverReq.Zone, serverReq.CommercialType)
+		serverType := getServerType(apiInstance, serverReq.Zone, serverReq.CommercialType)
```

Comment:
`serverType` shadows the one created in previous context
```suggestion
		serverType = getServerType(apiInstance, serverReq.Zone, serverReq.CommercialType)
```